cha-ching-app/src/Error.js

```diff
@@ -0,0 +1,13 @@
+import "./error.css";
+
+const Error = (message, code, type) => {
+  return (
+    <div className="errorWrapper">
+      <div className="errorContainer">
+        <div className="errorText">error</div>
+      </div>
+    </div>
+  );
+};
```

Comment:
Nice! It's okay to make this a basic component for now that is not yet reusable (meaning removing the params that are getting passed in). Later on we could make this a more dynamic component that can work for multiple cases.

Reply:
Sounds good! I will remove them for now. Thank you!